docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,97 @@
+==== Known issues
+
+* Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+
+[discrete]
+[[breaking-changes-8.8.0]]
+==== Breaking changes
+
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+* The privileges for attaching alerts to cases has changed. Now, you need at least `Read` privileges for Security and `All` privileges for Cases ({pull}147985[#147985]).
+* Rule API changes are introduced with conditional actions. In {elastic-sec} 8.7 and earlier, action frequencies were set on a rule level by defining the `throttle` field. In 8.8, action frequencies are set on an action level and the `throttle` field is replaced by the `frequency` and `alert_filters` fields. The following APIs are affected:
+** <<rules-api-get,Get rule>>
+** <<rules-api-find,Find rules>>
+** <<rules-api-create,Create rule>>
+** <<rules-api-update,Update rule>>
+** <<bulk-actions-rules-api,Bulk rule actions>>
+
+//end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.8.0]]
+==== Deprecations
+There are no deprecations in 8.8.0.
+
+
+[discrete]
+[[features-8.8.0]]
+==== New features
+
+* Adds a new response action that allows you to execute commands on a selected host (https://github.com/elastic/security-team/issues/5441[5441]).
+* Adds a threat intelligence overview section to alert details for alerts generated by indicator rules ({pull}155328[#155328]).
+* Adds the `kibana.alert.url` field to alert events. Its value is a link to the alert details in Kibana ({pull}155069[#155069]).
+* Adds an option to duplicate a shared exception list from its more actions menu ({pull}154991[#154991]).
+* Adds a button that allows you to delete a note from a Timeline ({pull}154834[#154834]).
+* Adds rule action filters that let you specify whether actions should only run under particular conditions ({pull}154680[#154680]).
+* Adds a "snooze rule" button to the rules table ({pull}153083[#153083]).
+* Adds a "snooze rule" button to the rule details and edit rule pages ({pull}155407[#155407], {pull}155612[#155612]).
+* Adds controls to the Alerts page that allow you to customize which filters appear at the top of the page ({pull}152450[#152450]).
+* Adds an expandable section to the alert details flyout with various additional features and information. NOTE: to access this functionality you currently need to add a line to the `kibana.json` file. {pull}152935[#152935], {pull}152767[#152767], {pull}152303[#152303], {pull}152150[#152150], {pull}152047[#152047], {pull}150240[#150240], {pull}153807[#153807], {pull}153187[#153187], {pull}153706[#153706], {pull}153074[#153074], {pull}154196[#154196], {pull}153903[#153903], {pull}154114[#154114]).
```

Comment:
If this does happen to go into 8.8, the note should be a separate paragraph so it can render properly. (But my $0.02 is the whole note should prob be deferred to 8.9 or 8.10 when the feature isn't behind a feature flag.)

Reply:
Cool I agree, best to wait until the feature flag is removed.